pkgs/by-name/az/azure-artifacts-credprovider/package.nix

```
@@ -0,0 +1,39 @@
+{ lib
+, buildDotnetModule
+, dotnetCorePackages
+, stdenv
+, libunwind
+, libuuid
+, icu
+, openssl
+, zlib
+, curl
+, dotnet-sdk_6
+, fetchFromGitHub
+}:
+buildDotnetModule rec {
+  inherit dotnet-sdk_6;
+  version = "1.0.9";
```

```suggestion
  version = "1.3.0";
```
New version released by now